<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="1" hostname="liqingMacBook-Pro.local" name="pytest" skipped="2" tests="4" time="22.789" timestamp="2020-03-25T23:47:02.151281"><testcase classname="test_case.test_weather_condition.TestWeatherCondition" file="test_case/test_weather_condition.py" line="27" name="test_weather_desc_equal" time="0.002"><skipped message="unconditional skip" type="pytest.skip">test_case/test_weather_condition.py:27: unconditional skip</skipped></testcase><testcase classname="test_case.test_weather_condition.TestWeatherCondition" file="test_case/test_weather_condition.py" line="39" name="test_temp_equal" time="0.000"><skipped message="unconditional skip" type="pytest.skip">test_case/test_weather_condition.py:39: unconditional skip</skipped></testcase><testcase classname="test_case.test_weather_condition.TestWeatherCondition" file="test_case/test_weather_condition.py" line="45" name="test_1" time="18.185"></testcase><testcase classname="test_case.test_weather_condition.TestWeatherCondition" file="test_case/test_weather_condition.py" line="48" name="test2" time="2.501"><failure message="assert 2 == 1
  -2
  +1">self = &lt;test_case.test_weather_condition.TestWeatherCondition object at 0x109b69f28&gt;

    def test2(self):
&gt;       assert 2 == 1
E       assert 2 == 1
E         -2
E         +1

test_case/test_weather_condition.py:50: AssertionError</failure></testcase></testsuite></testsuites>